ods_ci/run_robot_test.sh

```diff
@@ -251,32 +251,46 @@ if command -v yq &> /dev/null
                     then
                         oc_host=${api_server}
                     else
-                        oc_host=$(yq  e '.OCP_API_URL' "${TEST_VARIABLES_FILE}")
+                        oc_host=$(yq -er '.OCP_API_URL' "${TEST_VARIABLES_FILE}") || {
```

Comment:
I  believe this breaks on latest yq version. Please hold on mergin this change. Unfortunately poetry installs yq v3 which is old. We need to install yq from github.

Reply:
Hm, does `yq` from GitHub has any advantages for us? Trying locally to update my `yq` via `pip` - looks like this is the latest one available there. So I suppose they changed the way how they distribute this tool? Still, do we need it in newer version? :upside_down_face:

Reply:
yeah, there were singificant changes from v3 to v4. We leverage on some changes present in v4 for out code. You can check the `install_idp.sh` script. Plus, it's better to use more recent version of libraries rather than very old ones for sec reason

Reply:
Okay, I'll try to check and verify this works with newer version also. Note - I am not able to easily test this on Mac, so I'll probably ask somebody to do this for me then.

Reply:
what issue do you face?

Reply:
not having a Mac machine :wink:

Reply:
ah sorry, I thought you were seeing issues testing in on Mac.

Reply:
@bdattoma, could you, please, be a bit more specific on what breaking issue did you expect, Berto? I downloaded the latest `yq` release from [GitHub](https://github.com/mikefarah/yq/releases/tag/v4.35.2) one and looks that it behaves as expected on my Fedora machine. I updated commit message appropriately so you can check details there.

Do you have a Mac machine? Can you test it too?

Reply:
if I remember correctly, lines like `yq --inplace '.KEY=env(value)' <yaml_filepath>` wasn't working fine on v3. 
> looks that it behaves as expected on my Fedora machine


`yq -er '.OCP_API_URL' "${TEST_VARIABLES_FILE}"` does this work with latest version for you?

Reply:
Yeah, as I stated above - it works fine for me both with `3.2.3` and also with the latest `4.35.2`.

The reason that `--inplace` doesn't work is that such flag isn't available in `3.2.3` version.

Although, as you can see in the `run_robot_test.sh` script, we use the `yq` only to grab three variables from the `test-variables.yaml` file. Nothing more. So I don't think it breaks anything else here.

Reply:
talked offline, it's ok to keep the -r which has been introduced in newer versions of the lib